### PR TITLE
Azure docs

### PR DESCRIPTION
**Pull request by Itxaka**
Closes https://github.com/rancher-sandbox/cOS-toolkit/issues/474

### CODE REVIEW

**Review by mudler — APPROVED** [review]

File: content/en/docs/Creating derivatives/Packer/build_azure.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+The suggested approach is based on using Packer templates to customize the
+deployment and automate the upload and publish to Azure of cOS derivatives or cOS itself. For all the details
+and possibilities of Packer check the [official documentation](https://www.packer.io/guides/hcl).
+
+## Run the build with Packer
+
+Publishing an image in Azure based on top of the latest cOS Vanilla image is
+fairly simple. In fact, it is only needed to set the Azure credentials
+and run a `packer build` process to trigger the deployment and register the
+resulting snapshot as an image. In such case the latest cOS image will be
```
Comment (mudler) [inline content/en/docs/Creating derivatives/Packer/build_azure.md:29]: we should maybe underline that it is required to upload the image before this. And maybe link to the docs on how to upload it

Comment (Itxaka) [reply content/en/docs/Creating derivatives/Packer/build_azure.md:29]: it is already in the requirements section with a link to the appropriate doc.

Comment (mudler) [reply content/en/docs/Creating derivatives/Packer/build_azure.md:29]: my bad, I mean a link to https://rancher-sandbox.github.io/cos-toolkit-docs/docs/getting-started/booting/#importing-an-azure-image-manually

File: content/en/docs/Creating derivatives/Packer/build_azure.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  in Azure public cloud provider by using the cOS Vanilla image.
+---
+
+![](https://docs.google.com/drawings/d/e/2PACX-1vSqJWcFThP7K2HS551LCqs73l4ZncXElLjlbCvxY96Ga2Jbjnq79j-DEjaccUZvYEQyphWiDQc9flxk/pub?w=1223&h=691)
+
+Requirements:
+
+* Packer
+* Azure access keys with the appropriate roles and permissions
+* [A Vanilla image](../../../development/build_raw_images#azure)
```
Comment (mudler) [inline content/en/docs/Creating derivatives/Packer/build_azure.md:17]:
```suggestion
* [A Vanilla image](../../../getting-started/booting/#importing-an-azure-image-manually)
```

I would avoid to link to dev docs, as this is mostly user facing, no need to build cOS to get the Azure image anymore :)

Comment (Itxaka) [reply content/en/docs/Creating derivatives/Packer/build_azure.md:17]: wow, totally forgot that existed, was even thinking of creating docs on how to upload all the base cloud vanilla images :o

**Pull request merged**